src/Form/existing-login-detection.js

```diff
@@ -0,0 +1,197 @@
+//
```

Comment:
This whole file is copied over from the existing fireproofing implementation. I've just fixed linting errors and added a tiny bit of logic to also trigger the PM API when needed. This will be removed soon for a custom logic that will use our autofill heuristics for field detection, form scoring, etc.